openwisp_notifications/types.py

```diff
@@ -17,9 +17,26 @@
         'email_notification': True,
         'web_notification': True,
     },
+    'general_message': {
+        'level': 'info',
+        'verb': 'message verb',
+        'verbose_name': 'Message Type',
+        'email_subject': '[{site.name}] Message Notification Subject',
+        'message': (
+            'Message notification with {notification.verb} and level {notification.level}'
+            ' by [{notification.actor}]({notification.actor_link})'
+        ),
+        'description': '{notification.description}',
+        'message_template': 'openwisp_notifications/default_message.md',
```

Comment:
@pandafy: if we have `message_template` here, do we need to also define the `message` key above?
Isn't it redundant? If so, I would remove `message` as moreover it's not flagged as translatable.

Reply:
Yes, only one of them is required.